src/RestControllers/FHIR/FhirMetaDataRestController.php

```diff
@@ -36,6 +42,36 @@ class FhirMetaDataRestController
     private $fhirPatientService;
     private $fhirService;
     private $fhirValidate;
+    /**
+     * The SMART extension capabilites that our system supports
+     * @see http://hl7.org/fhir/smart-app-launch/conformance/index.html
+     *
+     * All of these capabilities for MU3 are required to be implemented before HIT certification
+     * can be complete.
+     * @see ONC final rule commentary https://www.federalregister.gov/d/2020-07419/p-1184 Accessed on December 9th 2020
+     */
+    const SMART_CAPABILITIES = array (
```

Comment:
I just realized I left these in, I'll be removing these as its now been moved to the Capability class